src/services/ExperienceService.ts

```
@@ -0,0 +1,190 @@
+        console.log("Experiences fetched successfully:", res);
+        return res.json();
+      } catch (error) {
+        console.error("Error fetching experiences:", error);
+        throw error;
+      }
+    }
+  
+    static async addExperience(token: string, experienceData: Omit<ExperienceDetail, 'id'>) {
+      try {
+        console.log("Adding experience with data:", experienceData);
+        const res = await fetch(`${this.BASE_URL}/experiences`, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)